### ISSUE TITLE
Reminder issue to work on structured data on Commons when it's rolled out :-)

### ISSUE THREAD

**Issue by SandraF-WMF** [yarl/pattypan#49]
Hey Yarl, I have created a [Phabricator task](https://phabricator.wikimedia.org/T181057) in parallel to this issue as well, to keep track of updates to Pattypan that will support Structured Commons in the future. It's still some time away, but good to be aware of it already. Let me know if I can help with something!

**Comment by Abbe98** [comment]
Superseded by #123
